src/containers/ReviewModal/index.jsx

```diff
@@ -1,76 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import {
   FullscreenModal,
-  Container,
+  Row,
+  Col,
 } from '@edx/paragon';
 
-import createDOMPurify from 'dompurify';
-import parse from 'html-react-parser';
-
 import selectors from 'data/selectors';
 import actions from 'data/actions';
-import thunkActions from 'data/thunkActions';
 
 import ReviewActions from './ReviewActions';
+import ResponseContainer from './ResponseContainer';
 
 import './ReviewModal.scss';
+import { GradingRubric } from './GradingRubric';
 
 /**
  * <ReviewModal />
  */
-export class ReviewModal extends React.Component {
-  constructor(props) {
-    super(props);
-    console.log('review modal');
-    console.log({ props });
-    this.purify = createDOMPurify(window);
-    this.onClose = this.onClose.bind(this);
-  }
-
-  get textContent() {
-    return parse(this.purify.sanitize(this.props.response.text));
+export const ReviewModal = ({
+  setShowReview,
+  oraName,
+  isOpen,
+}) => {
+  const [showGradingRubric, setshowGradingRubric] = useState(false)
+  const toggleShowGradingRubric = () => {
+    setshowGradingRubric(!showGradingRubric);
   }
 
-  onClose() {
-    this.props.setShowReview(false);
-  }
-
-  render() {
-    return (
-      <FullscreenModal
-        title={this.props.oraName}
-        isOpen={this.props.isOpen}
-        beforeBodyNode={<ReviewActions />}
-        onClose={this.onClose}
-      >
-        <Container size="md">
-          {this.textContent}
-        </Container>
-      </FullscreenModal>
-    );
-  }
+  return (
+    <FullscreenModal
+      title={oraName}
+      isOpen={isOpen}
+      beforeBodyNode={(
+        <ReviewActions onShowGradingRubricClick={toggleShowGradingRubric} showGradingRubric={showGradingRubric} />
+      )}
+      onClose={() => setShowReview(false)}
```

Comment:
Part of why this was a class component was to avoid embedding this callback this way, as this is more brittle to test.